r/R/install-arrow.R

```diff
@@ -190,6 +190,9 @@ reload_arrow <- function() {
 #' }
 #' @export
 create_package_with_all_dependencies <- function(dest_file = NULL, source_file = NULL) {
+  if (!requireNamespace("pkgdown", quietly = TRUE)) {
+    stop("Package \"pkgdown\" is required to build the source tarball.")
+  }
```

Comment:
```suggestion
  if (!requireNamespace("pkgbuild", quietly = TRUE)) {
    stop("Package \"pkgbuild\" is required to build the source tarball.")
  }
```

Seems typo?

Reply:
🤦‍♂️whoops, sorry about that copy/paste error. thanks for catching that.